src/php/config/php.ini

```diff
@@ -1769,11 +1769,16 @@ opcache.enable=1
 ; Determines if Zend OPCache is enabled for the CLI version of PHP
 ;opcache.enable_cli=0
 
+# Set default values for: 
+# opcache.memory_consumption
+# opcache.interned_strings_buffer
+# See https://github.com/nextcloud-snap/nextcloud-snap/issues/2024
+
 ; The OPcache shared memory storage size.
-;opcache.memory_consumption=128
+opcache.memory_consumption=128
 
 ; The amount of memory for interned strings in Mbytes.
-;opcache.interned_strings_buffer=8
+opcache.interned_strings_buffer=8
```

Comment:
Btw, just wanted to mention that when opcache.memory_consumption is set to the default of 128, setting opcache.interned_strings_buffer to 32 would always hide the warning. See https://github.com/nextcloud/server/pull/32902/files

Reply:
And based on my own observation is ir nearly impossible to make the opcache memory of 128m full with a nextcloud instance

Reply:
So what I wanto to say is when opcache.memory_consumption is set to 128 and opcache.interned_strings_buffer is set to 32, I am not sure if a config value is needed

Reply:
Totally missed this until now-- that's brilliant @szaimen, thank you!

Reply:
Proposed in #2301.

Reply:
LGTM 👍